src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -1927,6 +1611,7 @@ namespace OutdoorAirUnit {
 		for ( EquipNum = 1; EquipNum <= OutAirUnit( OAUnitNum ).NumComponents; ++EquipNum ) {
 			EquipType = OutAirUnit( OAUnitNum ).OAEquip( EquipNum ).ComponentType;
 			EquipName = OutAirUnit( OAUnitNum ).OAEquip( EquipNum ).ComponentName;
+//			if( OutAirUnit( OAUnitNum ).OAEquip( EquipNum ).MaxVolWaterFlow == AutoSize ) Sim = false;
 			SimOutdoorAirEquipComps( OAUnitNum, EquipType, EquipName, EquipNum, OutAirUnit( OAUnitNum ).OAEquip( EquipNum ).ComponentType_Num, FirstHVACIteration, OutAirUnit( OAUnitNum ).OAEquip( EquipNum ).ComponentIndex, Sim );
 		}
```

Comment:
I considered protecting ControlCompOutput, which would fail is max fluid flow was = -99999, but decided this is robust as is